tests/fixtures/HLS-VI.L30.T06WVS.2024120T211159.v2.0.cmr.xml

```diff
@@ -245,7 +245,7 @@
   </OnlineAccessURLs>
   <OnlineResources>
   </OnlineResources>
-  <DataFormat>Cloud Optimized GeoTIFF (COG)</DataFormat>
+  <DataFormat>COG</DataFormat>
```

Comment:
[self comprehension question / getting up to speed]

It looks like the list of values is enumerated in the CSV attachment, is that right?
https://wiki.earthdata.nasa.gov/display/CMR/Archive+and+Distribution+Information+for+Collections#ArchiveandDistributionInformationforCollections-BestPractices

Reply:
Correct